tests/utils/test_templates.py

```
@@ -0,0 +1,538 @@
+        assert (
+            "<working_representation>User enjoys painting and photography</working_representation>"
+            in result
+        )
+        assert (
+            "<recent_conversation_history>\n"
+            "User: I painted a landscape today\n"
+            "</recent_conversation_history>"
+        ) in result
+        assert "The query is about user alice." in result
+        assert (
+            "The user's known biographical information:\n" "Name: Alice\n" "Age: 28\n"
```

Ruff (0.14.3)

Implicitly concatenated string literals on one line

Combine string literals

(ISC001)